common-ktx/src/test/java/co/nimblehq/common/extensions/DateExtensionsTest.kt

```diff
@@ -0,0 +1,339 @@
+package co.nimblehq.common.extensions
+
+import org.junit.Assert.*
+import org.junit.Test
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
+ * Unit test for DateExtensions.kt.
+ */
+class DateExtensionsTest {
+
+    @Test
+    fun plus() {
```

Comment:
Please follow our [convention](https://nimblehq.co/compass/development/android/testing/#test-case-naming) on Compass when naming the test case 🙏

Reply:
The test cases should be `private` rather than default access modifier

Reply:
@hoangnguyen92dn UPdated